### DISCUSSION TITLE
[postponed][manuscript][outreach][trackfind] No way to backport from URI to CURIE in identifiers.org.

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#107] [Team Posts]
Do we need this?

**Comment by dzerbino** [comment]
Not necessarily, but it will be a minor hassle for people wishing to update their metadata. I have pointed this out to the identifiers.org team, who said they'd think about it.

**Comment by jmfernandez** [comment]
There might be one way. identifiers.org also includes URL patterns. Although they are there for other purposes (authoritative and alternate sources), we could use it

**Comment by sveinugu** [comment]
Ah. Nice tip. @dtitov This is something we could implement in the TrackFind curation mode, in case identifiers.org does not do anything with it. But this is anyway for later.

**Comment by sveinugu** [comment]
TODO: Michael something from identifiers.org reached out with a comment after the webinar that we should contact them and look for an solution.

**Comment by sveinugu** [comment]
Added links to this in https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/41 and https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/49. Set to postponed
